docs/openhabian.md

```diff
@@ -313,11 +313,11 @@ We will re-add the functionality as soon as the underlying issue is resolved.
 
 ##### openHAB Dashboard
 
-After the installation of openHABian was successful, you should be able to access the openHAB dashboard:
+After the installation of openHABian was successful, you should be able to access the openHAB dashboard
+with the following URLs:
 
-- Raspberry Pi image setup: [http://openhabianpi:8080](http://openhabianpi:8080)
-- Pine A64 image setup: [http://openhabianpine64:8080](http://openhabianpine64:8080)
-- In any case: [http://your-device-hostname:8080](http://your-device-hostname:8080) or [http://192.168.0.2:8080](http://192.168.0.2:8080) (replace name/IP)
+* [http://openhab:8080](http://openhab:8080)
+* e.g. [http://192.168.0.2:8080](http://192.168.0.2:8080) (replace IP accordingly)
```

Comment:
why a line break in the sentence above?

In the list I would suggest to prepend something like "hostname or DNS name:" and "IP:"